lib/pure/concurrency/threadpool.nim

```diff
@@ -230,12 +231,12 @@ proc nimFlowVarSignal(fv: FlowVarBase) {.compilerproc.} =
     signal(fv.cv)
 
 proc awaitAndThen*[T](fv: FlowVar[T]; action: proc (x: T) {.closure.}) =
-  ## Blocks until the ``fv`` is available and then passes its value
-  ## to ``action``.
+  ## Blocks until `fv` is available and then passes its value
+  ## to `action`.
   ##
-  ## Note that due to Nim's parameter passing semantics this
-  ## means that ``T`` doesn't need to be copied so ``awaitAndThen`` can
-  ## sometimes be more efficient than `^ proc <#^,FlowVar[T]>`_.
+  ## Note that due to Nim's parameter passing semantics, this
+  ## means that `T` doesn't need to be copied, so `awaitAndThen` can
+  ## sometimes be more efficient than the `^ proc <#^,FlowVar[T]>`_.
```

Comment:
nit: in many of your PR's you're adding an article `the` which IMO should be omitted when it refers to such kind of link.

Reply:
My version is the grammatically correct version though, isn't it?

Reply:
how about (for future such cases, I don't really care about this isolated case) doing:

```nim
## sometimes be more efficient than the `^ proc <#^,FlowVar[T]>`_.
```
=>
```nim
## sometimes be more efficient than `^<#^,FlowVar[T]>`_.
```

it's still grammatically correct, shorter, and doesn't make a distinction between the fact that this symbol is a macro/template/proc/func (a user interested in knowing whether the linked symbol is a macro/template/proc/func can just click it, or, maybe docgen could automatically insert the symbol kind). Note that API's should often be allowed to switch between those (eg proc => func or proc => template) transparently, so distinguishing this in doc links shouldn't be needed.

Reply:
Ok, I will keep that in mind.